algorithmic_efficiency/workloads/mnist/mnist_pytorch/workload.py

```diff
@@ -124,6 +117,7 @@ def init_model_fn(self, rng: spec.RandomState) -> spec.ModelInitState:
     self._param_shapes = {
         k: spec.ShapeTuple(v.shape) for k, v in model.named_parameters()
     }
```

Comment:
it doesn't really matter because it's just the MNIST one, but should we replace this with
```python
self._param_shapes = param_utils.pytorch_param_shapes(model)
```